### PR TITLE
De-google the docs

### PR DESCRIPTION
**Pull request by petert82**
I've tried to rephrase things as much as possible so it sounds a bit more shpock-BE-specific.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,21 @@
-# Google Engineering Practices Documentation
+# Shpock Backend Engineering Practices Documentation
```
Comment (oestoer) [inline README.md:1]: I can't add a comment to the License sections, but do you think as per the CC-By 3.0 terms we should maybe attribute the original work?

**Review by oestoer — COMMENTED** [review]
Thanks Peter, one thought...

File: README.md [hunk 2]
```diff
@@ -26,8 +25,12 @@ we clarify here for external readers:
 
 ## License
 
-The documents in this project are licensed under the
-[CC-By 3.0 License](LICENSE), which encourages you to share these documents. See
-<https://creativecommons.org/licenses/by/3.0/> for more details.
+The documents in this project are based on the [Google Engineering Practices][1]
+documentation. They are licensed under the [CC-By 3.0 License](LICENSE), which
+encourages you to share these documents.
```
Comment (petert82) [inline README.md:30]: > I can't add a comment to the License sections, but do you think as per the CC-By 3.0 terms we should maybe attribute the original work?

@oestoer: Added this. Do you think this should be sufficient?

Comment (oestoer) [reply README.md:30]: Thanks, I think it's good

**Review by oestoer — APPROVED** [review]
LGTM

**Pull request merged**